gloss/crt0.S

```diff
@@ -59,7 +66,7 @@ _start:
   addi t0, t0, 1
   j 1b
 1:
-  andi sp, sp, -16
+  addi sp, sp, -16
```

Comment:
Why `addi` instead of `andi`? `andi sp, sp, -16` align the stack to 16 bytes, but `addi sp, sp, -16` seems like just allocate 16 bytes stack?